spec/compass/config.rb

```diff
@@ -1,34 +1,28 @@
-http_path       = "/"
-css_dir         = "stylesheets"
-sass_dir        = "sass"
-images_dir      = "images"
-javascripts_dir = "javascripts"
+# rubocop:disable Lint/UselessAssignment
+sourcemap = true
+output_style = :compressed
```

Comment:
Could you explain why these two local variables matter for the Compass configuration test? It's bewildering to me how anything is using them since they aren't referenced.

Reply:
Maybe it is some old code. I will try to remove it.

Reply:
Fixed dfcb2d5